lib/dashboard_checker.py

```diff
@@ -5,7 +5,7 @@
 import sys
 import json
 
-# Uses pyQuARC 1.1.5
+# Uses pyQuARC 2.0
```

Comment:
Maybe 2.0.0 just to have the standard format for future reference.

Reply:
Yeah, it is not that important for code, but it is for bamboo, but hopefully next PI we'll do away with the command line altogether.

Reply:
Actually I'm going to try and remove these files as I don't think they are needed anymore